src/components/Comments.svelte

```diff
@@ -253,7 +253,7 @@
             class="r-h-12 r-w-12 r-rounded-full"
           />
           <div
-            class="r-bg-primary-50 dark:!r-bg-gray-700 r-flex-grow r-px-6 r-py-4 r-max-w-full r-overflow-scroll"
+            class="r-bg-primary-50 dark:!r-bg-gray-700 r-flex-grow r-px-6 r-py-4 r-max-w-full"
```

Comment:
```suggestion
            class="r-bg-primary-50 dark:!r-bg-gray-700 r-flex-grow r-min-w-0 r-px-6 r-py-4 r-max-w-full"
```
might fix the comment box size (#2)